cvmfs/cloud_testing/run.sh

```diff
@@ -242,6 +247,12 @@ devel_package=$(read_package_map    ${ctu}/pkgmap "$platform" 'devel'    )
 unittest_package=$(read_package_map ${otu}/pkgmap "$platform" 'unittests')
 config_packages="$(read_package_map ${otu}/pkgmap "$platform" 'config'   )"
 
+# if we are on mac then we have to run an special script
```

Comment:
Could you do this check one level higher in `ci-scripts/cvmfs/run_cloudtests.sh`. I believe this is the more natural place to do this distinction.